x-pack/plugins/index_management/public/application/sections/home/index_list/details_page/details_page_overview/data_stream_details.tsx

```diff
@@ -0,0 +1,154 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { FunctionComponent, ReactNode } from 'react';
+import { i18n } from '@kbn/i18n';
+import { EuiButton, EuiFlexGroup, EuiFlexItem, EuiIcon, EuiText, EuiTextColor } from '@elastic/eui';
+import { css } from '@emotion/react';
+import { euiThemeVars } from '@kbn/ui-theme';
+import { reactRouterNavigate } from '@kbn/kibana-react-plugin/public';
+import { SectionLoading } from '@kbn/es-ui-shared-plugin/public';
+
+import { FormattedMessage } from '@kbn/i18n-react';
+import { getDataStreamDetailsLink } from '../../../../../services/routing';
+import { getTemplateDetailsLink } from '../../../../../..';
+import { useLoadDataStream } from '../../../../../services/api';
+import { useAppContext } from '../../../../../app_context';
+import { humanizeTimeStamp } from '../../../data_stream_list/humanize_time_stamp';
+import { OverviewCard } from './overview_card';
+
+export const DataStreamDetails: FunctionComponent<{ dataStreamName: string }> = ({
+  dataStreamName,
+}) => {
+  const { error, data: dataStream, isLoading, resendRequest } = useLoadDataStream(dataStreamName);
+  const { history } = useAppContext();
+  const hasError = !isLoading && (error || !dataStream);
+  let contentLeft: ReactNode = (
+    <EuiFlexGroup gutterSize="xs" alignItems="baseline">
+      <EuiFlexItem grow={false}>
+        <EuiText
+          css={css`
+            font-size: ${euiThemeVars.euiFontSizeL};
```

Comment:
wouldnt be the same (and perhaps cleaner) to just use the `size="L"` from `EuiText` instead? 🤔

Reply:
There is no `"L"` for text sizes for some reason, only `["xs", "s", "m", "relative"]` 🤔

Reply:
that is soooo strange! whyyyy! Your changes lgtm in the mean time tho